test/expected/full.css

```diff
@@ -843,8 +843,8 @@ body {
 }
 .flex-1 {
   width: 8.333333333333334%;
-  -ms-flex-preferred-size: 8.333333333333334%;
-      flex-basis: 8.333333333333334%;
+  -ms-flex-preferred-size: auto;
+      flex-basis: auto;
```

Comment:
is this backwards compatible?

Reply:
Yes, that's what we do to fix IE11, flex basis is just another way to set the width